cmd/control/config.go

```diff
@@ -290,13 +291,20 @@ func validate(c *cli.Context) error {
 func inputBytes(c *cli.Context) ([]byte, error) {
 	input := os.Stdin
 	inputFile := c.String("input")
-	if inputFile != "" {
-		var err error
-		input, err = os.Open(inputFile)
-		if err != nil {
-			return nil, err
-		}
-		defer input.Close()
+	if inputFile == "" {
+		return nil, errors.New("input parameter can not be empty")
+	}
+	input, err := os.Open(inputFile)
```

Comment:
Do we also need `os.Open`? I think you just only need `ioutil.ReadFile` once.

Reply:
We have to do that, or we'll block IO